src/misc/string_utils.cpp

```diff
@@ -286,6 +286,24 @@ bool ends_with(const std::string_view str, const std::string_view suffix) noexce
 	return std::equal(suffix.rbegin(), suffix.rend(), str.rbegin());
 }
 
+std::string strip_prefix(const std::string& str, const std::string& prefix) noexcept
+{
+	if (starts_with(str, prefix)) {
+		return str.substr(prefix.size());
+	} else {
+		return str;
+	}
+}
+
+std::string strip_suffix(const std::string& str, const std::string& suffix) noexcept
+{
+	if (ends_with(str, suffix)) {
+		return str.substr(0, str.size() - suffix.size());
+	} else {
+		return str;
+	}
+}
+
```

Comment:
Suggest dropping the `else` and return directly. 
Then it become positive scenario versus default:

```c++
    if (ends_with(str, suffix)) {
        return str.substr(0, str.size() - suffix.size());
    }
    return str;
```

Reply:
SUre.